pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,17 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
 	for i := 0; i < inType.NumField(); i++ {
 		inField := inType.Field(i)
 		outField, found := outType.FieldByName(inField.Name)
 		if !found {
-			return fmt.Errorf("couldn't find a corresponding field %v in %v", inField.Name, outType)
+			// aggregate the errors so we can return them at the end but still provide
+			// best effort for generation for other fields in this type
+			errs = append(errs, fmt.Sprintf("couldn't find a corresponding field %v in %v", inField.Name, outType))
+			continue
+
+//			return fmt.Errorf("couldn't find a corresponding field %v in %v", inField.Name, outType)
```

Comment:
added so that branch can be pulled and commented/uncommented for test.  Should be removed if we keep this PR